napari/layers/pyramid/tests/test_pyramid.py

```diff
@@ -4,6 +4,10 @@
 from napari.layers import Pyramid
 
 
+# Set random seed for testing
+np.random.seed(0)
```

Comment:
Typically these get set within each function, because things like parallel running of tests will break this. I think there's a way to set it once using a fixture but I've never used that.

Reply:
ah should i move to every example?

Reply:
ideally yes, sorry :grimacing:

Reply:
I fear if you don't we will end up with more weird stochastic failures, and we have enough of those! :joy:

Reply:
it's ok - will do now